.github/actions/diffs/action.yml

```diff
@@ -32,6 +31,9 @@ runs:
           - '*.md'
           - '.github/workflows/docs.yml'
         isMove:
-          - 'crates/sui-framework/sources/**'
-          - 'crates/sui/src/sui_move/**'
+          - 'crates/sui-framework/**'
+          - 'crates/sui-framework-build/**'
+          - 'crates/sui-framework-tests/**'
+          - 'crates/sui_move/**'
           - 'Cargo.toml'
+          - 'sui_programmability/**'
```

Comment:
Curious why this used to be in isRust?

Reply:
We used to only have isRust and no isMove. isMove was added when we added Move Prover run to CI.
sui_programmability was added to isRust many months ago by you https://github.com/MystenLabs/sui/pull/5167